### DISCUSSION TITLE
Rescan Library

### DISCUSSION THREAD

**Discussion by renatosttas** [photoview/photoview#500] [Q&A and Troubleshooting]
Good day!

I am a new user of Photoview (to replace my Google library). I have around 100k pictures and seems that Photoview just stop scanning. Can I force a new scan for all my library?

**Comment by viktorstrate** [comment]
Can't you just click on "Scan All" to start the scanner again?

**Comment by renatosttas** [comment]
2021/09/02 19:02:18 Insert Album /photos/2011/Q3 /photos/2011/Q3, contains photo is true
2021/09/02 19:02:18 Failed to get ignore file, err = open /photos/2011/Q4/.photoviewignore: no such file or directory

Strange! The folder have pictures inside.... What I am doing wrong?

**Reply by darrepac** [reply]
It is not saying the opposite
